target/linux/ar71xx/files/arch/mips/ath79/mach-dap-1533-a1.c

```diff
@@ -0,0 +1,180 @@
+/*
+ * D-Link DAP-1533 rev. A1 support
+ *
+ * Copyright (c) 2017 Taras Gavrylenko <gavrylenko.taras@gmail.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License version 2 as published
+ *  by the Free Software Foundation.
+ */
+
+#include <linux/pci.h>
+#include <linux/phy.h>
+#include <linux/gpio.h>
+#include <linux/platform_device.h>
+#include <linux/ath9k_platform.h>
+#include <linux/ar8216_platform.h>
+
+#include <asm/mach-ath79/ar71xx_regs.h>
+
+#include "common.h"
+#include "dev-ap9x-pci.h"
+#include "dev-eth.h"
+#include "dev-gpio-buttons.h"
+#include "dev-leds-gpio.h"
+#include "dev-m25p80.h"
+#include "dev-spi.h"
+#include "dev-usb.h"
+#include "dev-wmac.h"
+#include "machtypes.h"
+
+#define DAP1533A1_GPIO_LED_GREEN_POWER	22
+#define DAP1533A1_GPIO_LED_RED_POWER	14
+#define DAP1533A1_GPIO_LED_WLAN			12
+#define DAP1533A1_GPIO_LED_WLAN_CLIENT	19
+
+#define DAP1533A1_GPIO_BTN_RESET		17
+#define DAP1533A1_GPIO_BTN_WPS			16
+#define DAP1533A1_GPIO_SW_RF_5GHz		18
+#define DAP1533A1_GPIO_SW_BRIDGE		11
+
+#define DAP1533A1_KEYS_POLL_INTERVAL	20	/* msecs */
+#define DAP1533A1_KEYS_DEBOUNCE_INTERVAL	(3 * DAP1533A1_KEYS_POLL_INTERVAL)
+
+#define DAP1533A1_MAC0_OFFSET			0x00FFA0
+#define DAP1533A1_MAC1_OFFSET			0x00FFB4
+#define DAP1533A1_PCIE_CALDATA_OFFSET	0x1000
+
+static struct gpio_led dap1533a1_leds_gpio[] __initdata = {
+	{
+		.name		= "d-link:green:power",
```

Comment:
Please, use "board:color:function" LED names (don't forget to update them in user space scripts).